features/application-mgt/org.wso2.carbon.identity.application.mgt.server.feature/resources/identity/authenticationTemplates/age.json

```diff
@@ -68,6 +68,10 @@
     "                Log.debug('User ' + context.currentKnownSubject.identifier + ' is under aged. Hence denied to login.');",
     "                sendError(errorPage, errorPageParameters);",
     "            }",
+    "        }, onFail: function (context) {",
```

Comment:
Can we skip this and handle it at the 
```
case FAIL_COMPLETED:
```
when there is no such function registered. 
i.e. the default behavior is to handle retry if retry is enabled.

So the script is simple for this case.